### PR TITLE
CI: Ignore expected failures in PRs

### PR DESCRIPTION
**Pull request by effigies**

### CODE REVIEW

**Comment by emdupre** [comment]
Thanks, @effigies ! 

I had tried explicitly opening a PR to see if I would be able to get green checks as an org member (since I can access repo / org secrets), but it seems like that didn't work. Not sure if it has for you ?

For now, though I think ignoring the errors is sensible 👍

File: publications.qmd [hunk 1]
```diff
@@ -11,30 +11,34 @@ format:
...
+
+    db = setup_db(configfile)
+
+    print(get_publications(
+            db.get_collection('publications'),
+        ))
+
+    print('\ \n\ \n')
+    print(f'*Updated: {date.today()} - Publication list created using [autocv](https://github.com/poldrack/academicdb)*')
+except Exception:
```
[the remark this answers is not in the document]
Comment (effigies) [reply publications.qmd:40]: `StdinNotImplementedError` shows up in CI. Not sure if there are others that could be raised, but it doesn't seem super important. We either want them raised (`master`) or not (PR), IMO.

**Comment by effigies** [comment]
> Not sure if it has for you ?

No, it doesn't work for forks. It did work in #7 because I created it as a branch here. I can be a bit more specific and disable the environment variable when the PR originates here. Will take a bit more digging to get there.

**Comment by effigies** [comment]
Okay, the environment variable is set here, but not in #22.

* #17: https://github.com/poldracklab/poldracklab.github.io/actions/runs/5181218230/jobs/9336422517
* #22: https://github.com/poldracklab/poldracklab.github.io/actions/runs/5181219440/jobs/9336425305

**Comment by effigies** [comment]
In case it was unclear, I think this is good to merge.

**Comment by poldrack** [comment]
done, thanks!

**Pull request merged**